application/src/main/java/org/opentripplanner/standalone/server/DefaultServerRequestContext.java

```diff
@@ -80,6 +84,7 @@ private DefaultServerRequestContext(
     VehicleParkingService vehicleParkingService,
     @Nullable EmissionsService emissionsService,
     @Nullable SorlandsbanenNorwayService sorlandsbanenService,
+    @Nullable GraphQLSchema schema,
```

Comment:
Good, the schema her is nullable, changing the Module should then work out of the box.